scriptmodules/emulators/aethersx2.sh

```diff
@@ -0,0 +1,539 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="aethersx2"
+rp_module_desc="PS2 emu - AetherSX2"
+rp_module_help="ROM Extensions: .iso .chd\n\nCopy your PS2 roms to $romdir/ps2"
+rp_module_licence="PROP"
+rp_module_repo="git https://github.com/retropieuser/aethersx2.git main"
+rp_module_section="exp"
+rp_module_flags="!all 64bit"
+
+function depends_aethersx2() {
+    local depends=(matchbox-window-manager xorg xserver-xorg-input-all mesa-vulkan-drivers)
+
+    getDepends ${depends[@]}
+}
+
+function sources_aethersx2() {
+    gitPullOrClone
+}
+
+function install_aethersx2() {
+    tar -xzvf AetherSX2-v1.5-3606.tar.gz -C "/opt/retropie/emulators/aethersx2"
+    chmod +x /opt/retropie/emulators/aethersx2/AetherSX2-v1.5-3606/usr/bin/aethersx2
+}
+
+function configure_aethersx2() {
+    mkRomDir "ps2"
+    
+    local launch_prefix
+    isPlatform "kms" && launch_prefix="XINIT-WM:"
+
+    addEmulator 0 "$md_id" "ps2" "$launch_prefix$md_inst/AetherSX2-v1.5-3606/usr/bin/aethersx2 -nogui %ROM%"
+    addSystem "ps2"
+
+    [[ "$md_mode" == "remove" ]] && return
+
+    moveConfigDir "$home/.config/aethersx2" "$md_conf_root/ps2/Config"
+    mkUserDir "$md_conf_root/ps2/Config"
+    
+    # Create the ps2 BIOs directory if it doesn't exist
+    if [ ! -d "$home/RetroPie/BIOS/ps2" ]; then
+        mkdir -p "$home/RetroPie/BIOS/ps2"
+    fi
+
```

Comment:
```suggestion
    chown -R $user:$user "$$home/RetroPie/BIOS/ps2"
```
i could not add any files to the folder due to it being owned by ``root``